account_asset_management/models/account_asset.py

```diff
@@ -441,9 +429,9 @@ def _onchange_purchase_salvage_value(self):
                 {"amount": self.depreciation_base, "line_date": self.date_start}
             )
 
-    @api.model
-    def create(self, vals):
-        asset = super().create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        asset = super().create(vals_list)
```

Comment:
This returns several records, so you have to take it into account.

Reply:
i think its ok now